app/src/main/java/org/thoughtcrime/securesms/conversation/ConversationActionBarView.kt

```diff
@@ -75,7 +81,7 @@ class ConversationActionBarView @JvmOverloads constructor(
     ) {
         this.delegate = delegate
         binding.profilePictureView.layoutParams = resources.getDimensionPixelSize(
-            if (recipient.isClosedGroupRecipient) R.dimen.medium_profile_picture_size else R.dimen.small_profile_picture_size
+            if (recipient.isClosedGroupV2Recipient) R.dimen.medium_profile_picture_size else R.dimen.small_profile_picture_size
```

Comment:
This should check for both legacy and v2 groups (using the new version of isGroupRecipient which needs to be added)